Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,76 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<int?> GetUserIdAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select IdUser from [User] u where u.Email = @Email;";
+            var queryParams = new { Email = accountName };
+            return await connection.QueryFirstOrDefaultAsync<int?>(query, queryParams);
+        }
+
+        public async Task UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesSettings)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+
+            const string updateQuery =
+                @"UPDATE [UserShippingLimit] SET Active = @Active, Interval = @IntervalInDays, Amount = @EmailsAmountByInterval WHERE IdUser = @IdUser;";
+            await connection.ExecuteAsync(updateQuery, contactPoliciesSettings
+            );
+        }
+
+        public async Task<List<ExcludedSubscribersLists>> VerifyUserSubscriberListExist(int? idUser,
+    List<ExcludedSubscribersLists> excludedSubscribersLists)
```

Comment:
I think that we do not need al `List`
```suggestion
    IEnumerable<ExcludedSubscribersLists> excludedSubscribersLists)
```